tests/main/interfaces-calendar-service/task.yaml

```diff
@@ -26,12 +26,10 @@ prepare: |
 restore: |
     kill "$(cat dbus-launch.pid)"
 
-    # In case the process gvfsd-metadata does not finish by itself, it is manually stopped
-    # The reason is that gvfsd-metadata locks the xdg/share/gvfs-metadata directory content
-    # producing an error when the xdg directory is removed.
-    if pid="$(pidof gvfsd-metadata)"; then
-        kill -9 "$pid" || true
-    fi
+    # We make sure the gvfsd-metadata process is stopped because it locks the 
+    # xdg/share/gvfs-metadata directory content producing an error when the 
+    # xdg directory is removed.
+    pkill gvfsd-metadata || true
```

Comment:
I think this should preserve KILL, so `pkill -9 gvfsd-metadata`.

Reply:
OTOH, this does not seem to address the problem. Rough guess, but `gvfsd-metadata` is probably writing to that directory. By the time `rm -rf ..` reaches `rmdir()` (and having removed entries that were inside), `gvfsd-metada` has probably created a new file inside, hence triggering `ENOTEMPTY`. Perhaps we ought to kill all gvfs processes instead, including `gvfsd`, so `pkill -9 gvfs` instead?

Reply:
Talked about this during standup yesterday. From digging into gvfs source code, it looks like all writes to metadata go through the gvfsd-metadata. That daemon is DBus activated. If there's any other gvfs process running (gvfsd or one of the monitors) they could try to access the metadata daemon and DBus will just resurrect it. Thus, I think a better solution is to make sure that the whole gvfs gang is stopped.

Reply:
@bboozzoo sure, I'll make that change. Thanks for reviewing!!